evap/results/templates/results_course_detail_questionnaires.html

```diff
@@ -12,7 +12,14 @@ <h4>{{ section.questionnaire.public_name }}</h4>
 <table class="table table-striped{% if not last %} mb-3{% endif %}">
     <tbody>
         {% for result in section.results %}
-            {% if result.question.is_rating_question %}
+            {% if result.question.is_heading_question %}
+    </tbody>
+</table>
```

Comment:
can you try out not reducing the indentation? or put some comments around this part so one can visually better understand what's going on?